js/main.js

```diff
@@ -1,4 +1,16 @@
 /* global data */
+if (data.entries.length === 0) {
+  var $createRow = document.createElement('div');
+  $createRow.setAttribute('class', 'row');
+  var $createEmptyContent = document.createElement('div');
+  $createEmptyContent.setAttribute('class', 'set-middle');
+  var $createContent = document.createElement('p');
+  $createContent.textContent = 'No entries have been recorded';
+  var $appendToEntries = document.querySelector('#entries');
+  $appendToEntries.appendChild($createRow);
+  $createRow.appendChild($createEmptyContent);
+  $createEmptyContent.appendChild($createContent);
```

Comment:
You don't have to DOM create this element, you can have it written into your HTML, then `querySelector` for it and just hide/show it as necessary